Formula/libmf.rb

```diff
@@ -0,0 +1,41 @@
+class Libmf < Formula
+  desc "open source tool for approximating an incomplete matrix."
+  homepage "https://www.csie.ntu.edu.tw/~cjlin/libmf/"
+  url "http://www.csie.ntu.edu.tw/~cjlin/libmf/libmf-2.01.zip"
+  sha256 "75bb98a4e8f1a318d7d07556583727b4e301759904570bf527972d964d34ef30"
+  patch :DATA
```

Comment:
We need patches to be submitted and accepted upstream to keep them in Homebrew. If it's platform dependent they should still take that patch if they want to support OS X. If they don't want to support OS X: we don't want this in Homebrew core. Thanks!